sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -134,4 +143,108 @@ export class StudentController extends BaseController {
   ): Promise<void> {
     await this.studentService.synchronizeFromUserInfo(userToken);
   }
+
+  @Get("/apply-PD-status")
+  async applyForPDStatus(@UserToken() userToken: IUserToken): Promise<void> {
+    // Get student details
+    const existingStudent = await this.studentService.getStudentByUserName(
+      userToken.userName,
+    );
+    if (!existingStudent) {
+      throw new NotFoundException(
+        `No student was found with the student name ${userToken.userName}`,
+      );
+    }
+    //check pd status in db, student should only allowed to check PD status once
+    // existingStudent?.StudentPDSentAt is set when student apply for PD Status first
+    // StudentPDStatus is null  before PD checker update status
+    // StudentPDStatus is true if PD Confirmed by ATBC
+    // StudentPDStatus is false if PD Denied by ATBC
+    // if student has a valid only, he/she should allow for a PD check
+    // if studentPDVerified is true from `SFASDB`, then no need to apply for PD to ATBC
+
+    if (
+      (existingStudent?.validSIN ?? false) &&
```

Comment:
I believe that this logic could be simplified, like below. It will validate if not null and true.
```
(existingStudent?.validSIN)
```
Besides that, it seems that there are many redundant checks, can we narrow it down to validSIN and some other one that dictates if the request was made or not?